exec/java-exec/src/test/java/org/apache/drill/exec/testing/ControlsInjectionUtil.java

```diff
@@ -74,7 +75,7 @@ public static void setControls(final UserSession session, final String controls)
 
     final OptionManager options = session.getOptions();
     try {
-      options.getAdmin().validate(opValue);
+      SystemOptionManager.getValidator(DRILLBIT_CONTROL_INJECTIONS).validate(opValue);
```

Comment:
This could just be a reference to the validator itself, as it is also public and static, rather than looking it up by name.